### PR TITLE
[AVM] Updated Readme's to support AVM transition - Part (2)

### PR DESCRIPTION
**Pull request by ahmadabdalla**
# Description

Second batch of modules for updating readme's to align to AVM direction.

Supporting PR #4067 4067

<details>
<summary>Issues that this PR fixes</summary>
Fixes #123
Fixes #456
</details>

-->

### Pipeline references

| Pipeline |
| - |
| |

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

## Checklist

- [ ] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (readme)
- [ ] I did format my code

### CODE REVIEW

File: modules/container-registry/registry/README.md [hunk 1]
```diff
@@ -839,3 +805,41 @@ module registry './container-registry/registry/main.bicep' = {
 
 </details>
 <p>
+
+
+## Notes
+
+### Parameter Usage: `imageRegistryCredentials`
```
Comment (AlexanderSehr) [inline modules/container-registry/registry/README.md:812]: This parameter doesn't exist (anymore)

File: modules/db-for-postgre-sql/flexible-server/README.md [hunk 2]
```diff
@@ -806,3 +664,149 @@ module flexibleServer './db-for-postgre-sql/flexible-server/main.bicep' = {
...
+      startIpAddress: '10.10.10.1' //Start IP address for the firewall rule. Must be IPv4 format
+      endIpAddress: '10.10.10.10' //End IP address for the firewall rule. Must be IPv4 format
+  }
+]
+```
+
+</details>
+<p>
+
+### Parameter Usage: `configurations`
```
Comment (AlexanderSehr) [inline modules/db-for-postgre-sql/flexible-server/README.md:724]: This is a child resource for which we have docs

File: modules/db-for-postgre-sql/flexible-server/README.md [hunk 3]
```diff
@@ -806,3 +664,149 @@ module flexibleServer './db-for-postgre-sql/flexible-server/main.bicep' = {
...
+          source: 'user-override' // user-override, dynamic, system-default
+          value: 'INFO' // Value of the configuration
+      }
+]
+```
+
+</details>
+<p>
+
+### Parameter Usage: `databases`
```
Comment (AlexanderSehr) [inline modules/db-for-postgre-sql/flexible-server/README.md:767]: This is a child resource for which we have docs

File: modules/container-registry/registry/README.md (hunk 1 above)
Comment (ahmadabdalla) [reply modules/container-registry/registry/README.md:812]: removed

File: modules/db-for-postgre-sql/flexible-server/README.md (hunk 2 above)
Comment (ahmadabdalla) [reply modules/db-for-postgre-sql/flexible-server/README.md:724]: removed

File: modules/db-for-postgre-sql/flexible-server/README.md (hunk 3 above)
Comment (ahmadabdalla) [reply modules/db-for-postgre-sql/flexible-server/README.md:767]: removed

**Review by AlexanderSehr — APPROVED** [review]

**Pull request merged**